Makd.mak

```diff
@@ -651,11 +651,10 @@ d2conv: $O/d2conv.stamp
 
 $O/d2conv.stamp: $C
 	$Vfind $C -type f -regex '^.+\.d$$' > $@
-ifeq "$(shell d1to2fix --help | grep -- --input)" ""
+	@if d1to2fix --help | grep -- --input; then \
+	$(call exec, d1to2fix $(DIMPORTPATHS) --fatal --input=$@) \
+	@else \
```

Comment:
Something is very fishy about this, because you should also see the output of the `d1to2fix --help | grep -- --input` command. I'll test.